doc/releases/changelog-dev.md

```diff
@@ -37,34 +91,42 @@ Users can specify the control wires as well as the values to control the operati
     [Exp(1.570795j, 'PauliY', wires=[0])]
   ```
 
-  Moreover, the obtained tapered operation can be directly used within a QNode:
+  Moreover, the obtained tapered operation can be used directly within a QNode.
 
   ```pycon
     >>> dev = qml.device('default.qubit', wires=[0, 1])
     >>> @qml.qnode(dev)
     ... def circuit(params):
     ...     tap_op(params[0])
-    ...     return qml.expval(qml.PauliZ(0)@qml.PauliZ(1))
+    ...     return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
     >>> drawer = qml.draw(circuit, show_all_wires=True)
     >>> print(drawer(params=[3.14159]))
         0: ─Exp(1.570795j PauliY)─┤ ╭<Z@Z>
         1: ───────────────────────┤ ╰<Z@Z>
-
   ```
 
-<h4>Pauli Module</h4>
+<h4>(TODO: title) Pauli Module</h4>
 
-* Re-organized and grouped all functions in PennyLane responsible for manipulation of Pauli operators into a `pauli` 
+* Reorganized and grouped all functions in PennyLane responsible for manipulation of Pauli operators into a `pauli` 
```

Comment:
What is the impact this has on users?

Reply:
So this was the first PR in two which introduce the `PennyLane/pauli` module. In this PR I just made the folder and copied over files from the groping module, then reorganize them there. The only impact on a user would be that they can now find the functions in `qml.pauli.func_name`